### DISCUSSION TITLE
SCSS Modules

### DISCUSSION THREAD

**Discussion by kogan007** [remix-run/remix#6282] [Q&A]
Hey everyone, is there a way to extend the css module plugin to allow scss modules? I'm okay with patch package solutions.

**Comment by markdalgleish** [comment]
We also have a discussion tracking support for Sass more broadly if you want to add to the conversation there: https://github.com/remix-run/remix/discussions/5432

**Comment by wingleung** [comment]
Wanted this as well so I created a proof of concept within remix 
- https://github.com/remix-run/remix/pull/7612
